pkg/cluster/config/types.go

```diff
@@ -74,4 +80,6 @@ const (
 	// WARNING: this node type is not yet implemented!
 	// Please note that `kind` nodes hosting external load balancer are not kubernetes nodes
 	ExternalLoadBalancerRole NodeRole = "external-load-balancer"
+	// Default path where the CNI configuration(s) are placed.
+	CNIConfigPath = "/etc/cni/net.d/"
```

Comment:
this probably needs separate windows / linux constants.
having said that i have no idea what the recommended path for windows is.
we can ask in #sig-windows on slack.